docs/modules/ROOT/pages/custom-resolvers.adoc

```diff
@@ -185,6 +185,7 @@ const resolvers = {
 ----
 
 Instead, the following resolvers definition would be required:
```

Comment:
I can't seem to comment on it but there's a line above here that looks like this:

> The following resolvers definition would be invalid:

Should we change it to this: `The following resolvers definition would cause a warning to be raised:`?

Reply:
Good spot, fixed!